client/my-sites/plans/jetpack-plans/product-lightbox/product-details.tsx

```diff
@@ -0,0 +1,49 @@
+import { useMobileBreakpoint } from '@automattic/viewport-react';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import FoldableCard from 'calypso/components/foldable-card';
+import { SelectorProduct } from '../types';
+import DescriptionList from './description-list';
+
+type ProductDetailsProps = {
+	product: SelectorProduct;
+};
+
+const ProductDetails: React.FC< ProductDetailsProps > = ( { product } ) => {
+	const isMobile = useMobileBreakpoint();
+	const translate = useTranslate();
+	const [ expandedDetailType, setExpandedDetailType ] = useState( '' );
+
+	const productDetails = [
+		{ type: 'includes', title: translate( 'Includes' ), items: product.whatIsIncluded },
+		{ type: 'benefits', title: translate( 'Benefits' ), items: product.benefits },
+	];
+
+	return (
+		<>
+			{ productDetails.map( ( { type, title, items }, index, infoList ) => (
+				<div className="product-lightbox__detail-list" key={ type }>
+					{ isMobile ? (
+						<FoldableCard
+							hideSummary
+							header={ title }
+							clickableHeader={ true }
```

Comment:
Can we also pass this prop (`clickableHeader={ true }`) to `FoldableCard` in `client/my-sites/plans/jetpack-plans/product-store/features-list/index.tsx`?